File: src/main/java/com/chatsummary/bot/ChatSummaryBotApplication.java
package com.chatsummary.bot;

import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.retry.annotation.EnableRetry;
import org.springframework.scheduling.annotation.EnableScheduling;

@Slf4j
@SpringBootApplication
@EnableScheduling
@EnableRetry
public class ChatSummaryBotApplication {

    static void main(String[] args) {
        log.info("Starting ChatSummaryBot Application...");
        SpringApplication.run(ChatSummaryBotApplication.class, args);
    }
}

File: src/main/kotlin/com/chatsummary/bot/ChatSummaryBotApplication.kt
package com.chatsummary.bot

import io.quarkus.runtime.Quarkus
import io.quarkus.runtime.annotations.QuarkusMain
import org.slf4j.LoggerFactory

@QuarkusMain
object ChatSummaryBotApplication {
    private val logger = LoggerFactory.getLogger(ChatSummaryBotApplication::class.java)

    @JvmStatic
    fun main(args: Array<String>) {
        logger.info("Starting ChatSummaryBot Application...")
        Quarkus.run(*args)
    }
}
